tests/test_jointcal_decam.py

```diff
@@ -105,6 +105,53 @@ def test_jointcalTask_2_visits_constrainedAstrometry_no_photometry(self):
 
         self._testJointcalTask(2, relative_error, self.dist_rms_absolute, pa1, metrics=metrics)
 
+    def setup_jointcalTask_2_visits_constrainedPhotometry_no_astrometry(self):
+        """Help keep the constrainedPhotometry tests consistent and make
+        the differences between them more obvious.
+        """
+        self.config = lsst.jointcal.jointcal.JointcalConfig()
+        self.config.astrometryRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
+        self.config.photometryRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
+        self.config.photometryModel = "constrained"
+        self.config.sourceSelector['astrometry'].badFlags.append("base_PixelFlags_flag_interpolated")
+        self.config.doAstrometry = False
+        self.jointcalStatistics.do_astrometry = False
+
+        pa1 = 0.11
+        metrics = {'collected_photometry_refStars': 4865,
```

Comment:
Some comment about where all these numbers come from would be helpful.

Reply:
They're all emperical, once I've gotten the code to actually work. That's true for all of the `metrics` blocks in jointcal's tests because I don't know the correct result a priori, except for the `cfht_minimal` test which was designed so that I could manually compute the correct answer.

I have a note in the readme about this: https://github.com/lsst/jointcal/blob/master/Readme.md

Do you think I should include a similar note along with each of these blocks?

Reply:
Since the answer is slightly involved, I'd suggest you reference the Readme, or include that disclaimer once at the top of the file and reference that in tests with empirical test values.